layouts/shortcodes/security-products/suppressions-granular-access.md

```diff
@@ -0,0 +1,17 @@
+By default, all users have full access to [suppressions][201]. To use granular access controls to limit the roles that may edit a suppression rule:
```

Comment:
```suggestion
By default, all users have read and write access to [suppressions][201]. To use granular access controls to limit the roles that may edit a suppression rule:
```

Reply:
Suggesting "read and write" to be more specific as to what full access entails. (There may be a more accurate way to describe it though, e.g., "view and edit" vs. "read and write".)

Reply:
The UI uses "full access", but it makes sense to be more specific. Since the UI says view and edit in the permissions dialog box, I'll go with that.